models/task/task.go

```diff
@@ -163,28 +164,28 @@ func (u *TaskStruct) Save(repo *repository.RepositoryStruct, OauthToken *sql.Nul
 	if err != nil {
 		logging.SharedInstance().MethodInfo("task", "Save", true).Errorf("commit error:%v", err)
 		transaction.Rollback()
-		return false
+		return err
 	}
 	u.ID, _ = result.LastInsertId()
 	logging.SharedInstance().MethodInfo("task", "Save").Debugf("new task saved: %+v", u)
-	return true
+	return nil
 }
 
-func (u *TaskStruct) Update(repo *repository.RepositoryStruct, OauthToken *sql.NullString) bool {
+func (u *TaskStruct) Update(repo *repository.RepositoryStruct, OauthToken *sql.NullString) error {
```

Comment:
__[golint]__
* [golint] exported method TaskStruct.Update should have comment or be unexported